dev/attest/src/testTsVersions.ts

```diff
@@ -0,0 +1,28 @@
+import { rmSync, writeFileSync } from "node:fs"
+import { join } from "node:path"
+import { exit } from "node:process"
+import { fromPackageRoot, readPackageJson } from "../../runtime/fs.js"
+import { shell } from "../../runtime/shell.js"
+;(() => {
+    const versions: { [k: string]: string } = {
+        "4.8": "16.0.0"
+    }
+    const filePath = join(fromPackageRoot(), ".npmrc")
+    rmSync(filePath, { force: true })
+    writeFileSync(filePath, "save=false")
+    let exitCode = 0
+    for (const [tsVersion, tsMorphVersion] of Object.entries(versions)) {
+        shell(`pnpm i ts-morph@${tsMorphVersion}`)
+        console.log(`testing ${tsVersion}`)
```

Comment:
If this is permanent let's update it to something more explicit like `Testing against typescript@${tsVersion}...`